Package.swift

```diff
@@ -21,6 +21,23 @@ let package = Package(
             name: "Wire",
             path: "wire-library/wire-runtime-swift/src/main/swift"
         ),
+        // The tests depend on valid protos via gradle
```

Comment:
By defining these test targets most if not all Swift development can be done / validated in Xcode